Xamarin.Forms.Platform.iOS/Forms.cs

```diff
@@ -194,9 +194,9 @@ class IOSPlatformServices : IPlatformServices
 			public IOSPlatformServices()
 			{
 #if __MOBILE__
-				//The standard accisibility size for a font is 18, we can get a
+				//The standard accisibility size for a font is 17, we can get a
```

Comment:
fix the typo while you're at it...

is there a reference doc somewhere for this ? if so, please add a comment